### ISSUE TITLE
Component error: ptz_camera - Integration 'ptz_camera' not found.

### ISSUE THREAD

**Issue by Clebena** [fjramirez1987/PTZ-YCC365#10]
Everything was working perfectly but I m getting this message now
Component error: ptz_camera - Integration 'ptz_camera' not found.

I guess it has to do with an update but have no clue

**Comment by juanimoni** [comment]
Hi Clebena, it seems you have no access to the ptz_camera integration....
copy (again) the 3 files (services.yaml/manifest.json/__init__.py) to your folder config/custom_components/ptz_camera

**Comment by juanimoni** [comment]
... and remember to reboot Home Assistant...

**Comment by Clebena** [comment]
Yes I've tried this without success. Only thing that worked was to downgrade HA.

**Comment by lumbarhopkins** [comment]
I am a complete noob here, but I believe that the integration needs to be updated to include a manifest.json file. I got it working by creating the following:
{
    "domain": "ptz_camera",
    "name": "ptz_camera",
    "version": "0.0.1",
    "documentation": "https://www.example.com",
    "dependencies": [],
    "codeowners": [],
    "requirements": [],
    "iot_class": "local_push"
}

**Comment by lumbarhopkins** [comment]
I am a complete noob here, but I believe that the integration needs to be updated to include a manifest.json file. I got it working by creating the following:
{
"domain": "ptz_camera",
"name": "ptz_camera",
"version": "0.0.1",
"documentation": " https://www.example.com ",
" dependencies ": [],
" codeowners ": [],
" requirements ": [],
" iot_class ":" local_push "
}

**Comment by unmesh59** [comment]
I had this working last year, had stopped using it and recently tried to use it again with 2021.9.7 and the new ptz_camera code with manifest.json and I get this error message.

Anyone else have it working with a recent version of HA?

**Comment by malloc32** [comment]
Same problem :(

**Comment by eerison** [comment]
for me too :'( how did you pass the authentication on request? 
@juanimoni
